backend/index.js

```diff
@@ -1,9 +1,17 @@
+//ENV Configuration
+require("dotenv").config();
+
 // SERVER
 const server = require("./api/server.js");
+const connectDB = require("./config/connectDB");
 
 // PORT
 const PORT = process.env.PORT || 9001;
 
+// Connect to MongoDB
+// Must whitelist IP address for those connecting to the database
```

Comment:
What steps do we have to take to not need to whitelist IPs? That might be a pretty big hassle even in the development stage.

Reply:
We can whitelist everyone's ip address individually or we can allow access to anyone.

Reply:
Okay we'll be whitelisting IPs for now unless something changes

Reply:
Does Heroku allow ssh into an app instance? If so then you can "ssh tunnel" to the DB via the app instance. (or called ssh port forward) If this is an option dont ssh with a PWD only use ssh keys